twisted/conch/client/default.py

```diff
@@ -230,10 +239,10 @@ def getPrivateKey(self):
             return defer.succeed(keys.Key.fromFile(file))
         except keys.EncryptedKeyError:
             for i in range(3):
-                prompt = "Enter passphrase for key '%s': " % \
-                    self.usedFiles[-1]
+                prompt = "Enter passphrase for key '%s': " % self.usedFiles[-1]
                 try:
-                    p = self._getPassword(prompt)
+                    p = self._getPassword(prompt).encode(
+                        sys.getfilesystemencoding())
```

Comment:
This is totally optional feedback, since it's a private method, but I think this belongs in `_getPassword`.